packages/manager/apps/dedicated/client/app/dedicated/server/server.service.js

```diff
@@ -1505,10 +1505,7 @@ export default class ServerF {
           head(filter(plan.productName.split('-'), (ele) => /^\d+$/.test(ele))),
         );
         assign(plan, { bandwidth });
-
-        if (bandwidth !== existingBandwidth) {
-          return plan;
-        }
+        return plan;
```

Comment:
```suggestion
        return {
          ...plan,
          bandwidth
        }
```
that way we can  remove call to `_.assign`